core/lib/protobuf_config/src/proto/config/wallets.proto

```diff
@@ -5,6 +5,7 @@ package zksync.config.wallets;
 message PrivateKeyWallet {
   optional string address = 1; // optional
   optional string private_key = 2; // required
+  optional bool ignore_private_key = 3; // optional, this is hack for gkms_signer
```

Comment:
Does not look super clean.

It would be better if we could skip the private key check if mode = GKMS
or move the mode parameters in the wallets config if not accessible?